clients/vault/src/system.rs

```diff
@@ -196,8 +196,8 @@ fn parse_collateral_and_amount(
 
 #[derive(Parser, Clone, Debug)]
 pub struct VaultServiceConfig {
-	#[clap(long, help = "The Stellar secret key that is used to sign transactions.")]
-	pub stellar_vault_secret_key_filepath: String,
+	#[clap(long, help = "The filepath where the json config for StellarOverlay is located")]
+	pub stellar_overlay_config_filepath: String,
```

Comment:
Ah just one more thing I realized @b-yap. 
I did not really consider that this config replaces the vault secret key file. It's a good thing I guess, but then we need to change the README of the vault because it still contains the old parameter. And I would maybe change the internal structure of the config file so that the "secret_key" field is at the root level of the json file and not just contained in the "connection_info". Putting it in the root level should make it more apparent that this key is super important.